app/src/main/res/values/strings.xml

```diff
@@ -108,7 +108,7 @@
     <!-- Browser menu button to put the the current page in reader mode -->
     <string name="browser_menu_read">Reader view</string>
     <!-- Browser menu button to open the current page in an external app -->
-    <string name="browser_menu_open_app_link">Open in App</string>
+    <string name="browser_menu_open_app_link">Open in app</string>
```

Comment:
I'm pretty sure we need a new string resource here for localization to work properly :/ 
like we did in https://github.com/mozilla-mobile/fenix/pull/9247

Reply:
@ekager I initially thought so too, but as per the discussion [here](https://github.com/mozilla-l10n/android-l10n/pull/185), it seems we don't